src/llamafactory/model/loader.py

```diff
@@ -103,12 +106,13 @@ def load_tokenizer(model_args: "ModelArguments") -> "TokenizerModule":
         setattr(processor, "video_resolution", model_args.video_resolution)
         setattr(processor, "video_fps", model_args.video_fps)
         setattr(processor, "video_maxlen", model_args.video_maxlen)
-    except Exception:
+    except Exception as e:
+        logger.warning("Failed to load processor. Error: {}".format(e))
```

Comment:
We do not expect to raise a warning if the model is a text-only model

Reply:
the exception may be raised when it failed to download model from hf.
it is better to log the exception

Reply:
sorry we do not totally agree with this opinion